TranquilApp/ViewControllers/NotificationViewController.swift

```diff
@@ -22,29 +29,88 @@ class NotificationViewController: UIViewController {
+        background.didMove(toParent: self)
+        background.modalPresentationStyle = .fullScreen
+        view.insertSubview(background.view, at: 0)
+    }
+    
+    @objc private func addBreathingButton() {
+        breathingButton.setTitle("Press to go from notification to Breathing Exercises", for: .normal)
+        breathingButton.titleLabel?.numberOfLines = 0 // 0 indicates dynamic
+        breathingButton.titleLabel?.lineBreakMode = NSLineBreakMode.byWordWrapping
+        breathingButton.titleLabel?.textAlignment = .center
+        view.addSubview(breathingButton)
+        breathingButton.backgroundColor = UIColor(SECONDARY_TEXT_COLOR)
+        breathingButton.setTitleColor(.black, for: .normal)
+        breathingButton.frame = CGRect(x: 35, y:200, width: 330, height: 150)
+        breathingButton.layer.cornerRadius = 10
+        breathingButton.addTarget(self, action: #selector(didTapBreathingButton), for: .touchUpInside)
+    }
+
+    @objc private func addJournalButton() {
+        journalButton.setTitle("Press to go from notification to Journal", for: .normal)
+        journalButton.titleLabel?.numberOfLines = 0 // 0 indicates dynamic
+        journalButton.titleLabel?.lineBreakMode = NSLineBreakMode.byWordWrapping
+        journalButton.titleLabel?.textAlignment = .center
+        view.addSubview(journalButton)
+        journalButton.backgroundColor = .lightGray
+        journalButton.setTitleColor(.black, for: .normal)
+        journalButton.frame = CGRect(x: 35, y:375, width: 330, height: 150)
+        journalButton.layer.cornerRadius = 10
+        journalButton.addTarget(self, action: #selector(didTapJournalButton), for: .touchUpInside)
     }
     
-    @objc private func didTapButton() {
+    @objc private func addAiChatbotButton() {
+        aiChatbotButton.setTitle("Press to go from notification to ChatBot", for: .normal)
+        aiChatbotButton.titleLabel?.numberOfLines = 0 // 0 indicates dynamic
+        aiChatbotButton.titleLabel?.lineBreakMode = NSLineBreakMode.byWordWrapping
+        aiChatbotButton.titleLabel?.textAlignment = .center
+        view.addSubview(aiChatbotButton)
+        aiChatbotButton.backgroundColor = UIColor(SECONDARY_TEXT_COLOR)
+        aiChatbotButton.setTitleColor(.black, for: .normal)
+        aiChatbotButton.frame = CGRect(x: 35, y:550, width: 330, height: 150)
+        aiChatbotButton.layer.cornerRadius = 10
+        aiChatbotButton.addTarget(self, action: #selector(didTapAIChatbotButton), for: .touchUpInside)
+    }
+    @objc private func didTapBreathingButton() {
         let vc = BreathingViewController()
         vc.modalPresentationStyle  = .popover
         navigationController?.pushViewController(vc, animated: true)
-//        navigationController?.popViewController(animated: true)
-//        present(vc, animated: true)
+    }
+    
+    @objc private func didTapJournalButton() {
```

Comment:
running into a bug where it pushes the journal view up a bit...
<img width="269" alt="Screenshot 2023-03-01 at 2 36 44 PM" src="https://user-images.githubusercontent.com/44285485/222281436-018fcd82-a44b-4357-af4a-e2ca41b30482.png">